tidb-cloud/integrate-with-vercel.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Integrate with Vercel
+summary: Learn how to connect your TiDB Cloud clusters to Vercel projects.
+---
+
+# Integrate with Vercel
+
+## Introduction
+
+This guide will walk you through connecting your TiDB Cloud clusters to Vercel projects.
+
+It will cover two options:
+
+* Connecting your TiDB Cloud database to your Vercel projects by manually adding environment variables
+* Connecting your TiDB Cloud database to your Vercel projects by using the Vercel integration
+
+## Considerations
+
+### Vercel Accounts and Projects 
+
+The integration with Vercel requires that you have an account and a project in Vercel. If you don't have any, refer to
+
+* [Create a new personal account](https://vercel.com/docs/concepts/personal-accounts/overview) or [create a new team](https://vercel.com/docs/concepts/teams/overview) in Vercel.
+* [Create a new project](https://vercel.com/docs/concepts/projects/overview) in Vercel.
+
+A Vercel project can only be connected to one TiDB Cloud cluster. To change the integration, you need to first disconnect the current cluster and then connect to a new cluster.
+
+### TiDB Cloud Accounts and Clusters
+
+The integration with Vercel requires that you have an account and a cluster in TiDB Cloud. If you don't have any, refer to [create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md) in TiDB Cloud.
+
+To integrate with Vercel, you must have the `Organization Owner` access to your organization or `Project Member` access to the target project in TiDB Cloud.
+
+A TiDB Cloud cluster can connect to more than one Vercel projects.
+
+### TiDB Cloud Cluster Traffic Filter
+
+Vercel deployments use [dynamic IP addresses](https://vercel.com/guides/how-to-allowlist-deployment-ip-address).
+
+To connect to a TiDB Cloud cluster, the traffic filter of your TiDB Cloud cluster must allow all IP addresses (0.0.0.0/0). If TiDB Cloud doesn't find an entry for 0.0.0.0/0 in your cluster's traffic filters, TiDB Cloud adds it on your behalf, as part of the integration workflow.
+
+## Connect via the TiDB Cloud Vercel Integration
+
+You can install the [TiDB Cloud integration](https://vercel.com/integrations/tidb-cloud-dev) from the [Vercel's Integrations Marketplace](https://vercel.com/integrations). You can choose which cluster you want to connect to, and we'll automatically generate all the necessary environment variables into your Vercel projects.
+
+1. Click **Add Integration** button on the [TiDB Cloud integration page](https://vercel.com/integrations/tidb-cloud-dev).
+2. Choose the scope of your integration (personal or team).
+3. Select Vercel projects to connect to a TiDB Cloud cluster.
+4. Confirm the required permissions for integration.
+5. On the left side, select the Vercel projects you want to connect to.
```

Comment:
Better to have screenshots.

Reply:
Agree. But all other TiDB Cloud docs use pure text steps, I follow the status quo. Let's doc team give a suggestion.